DXMainClient/Domain/Multiplayer/CnCNet/MapSharer.cs

```diff
@@ -380,12 +384,21 @@ private static string DownloadMain(string sha1, string myGame, string mapName, o
 
             using (TWebClient webClient = new TWebClient())
             {
+                // TODO enable proxy support for some users
                 webClient.Proxy = null;
 
+                if (string.IsNullOrWhiteSpace(ClientConfiguration.Instance.CnCNetMapDBDownloadURL))
+                {
+                    success = false;
+                    return "MapSharer: Download URL is not set.";
```

Comment:
Pretty sure you want to log that and return `null` instead.

Reply:
Also let's use "configured" word instead of "set" again. The point is that configuration has more obvious meaning of something that can't be done in command, whereas setting could potentially be done in a map command.

Reply:
yeah, it means `return ex.Message;` this line must be incorrect